src/libcore/num/f32.rs

```diff
@@ -239,7 +239,9 @@ impl Float for f32 {
     /// Converts to degrees, assuming the number is in radians.
     #[inline]
     fn to_degrees(self) -> f32 {
-        self * (180.0f32 / consts::PI)
+        // Use a constant for better precision.
+        let pis_in_180 = 57.2957795130823208767981548141051703_f32;
```

Comment:
Why not an actual `const`?

Reply:
Yeah, that makes more sense stylistically.